app/enterprise/1.5.x/kong-for-kubernetes/install-on-kubernetes.md

```diff
@@ -0,0 +1,306 @@
+In the following steps, replace `<your-password>` with a secure password.
+
+{% navtabs %}
+{% navtab kubectl %}
+
+1. Create a sessions config file for Kong Manager:
+    ```
+    $ echo '{"cookie_name":"admin_session","cookie_samesite":"off","secret":"<your-password>","cookie_secure":false,"storage":"kong"}' > admin_gui_session_conf
+    ```
+2. Create a sessions config file for Kong Dev Portal:
+    ```
+    $ echo '{"cookie_name":"portal_session","cookie_samesite":"off","secret":"<your-password>","cookie_secure":false,"storage":"kong"}' > portal_session_conf
+    ```
+3. Create secret:
+    ```
+    kubectl create secret generic kong-session-config -n kong --from-file=admin_gui_session_conf=<manager-config-filename> --from-file=portal_session_conf=<portal-config-filename>
+    ```
+
+{% endnavtab %}
+{% navtab OpenShift oc %}
+
+1. Create a sessions config file for Kong Manager:
+    ```
+    $ echo '{"cookie_name":"admin_session","cookie_samesite":"off","secret":"<your-password>","cookie_secure":false,"storage":"kong"}' > admin_gui_session_conf
+    ```
+2. Create a sessions config file for Kong Dev Portal:
+    ```
+    $ echo '{"cookie_name":"portal_session","cookie_samesite":"off","secret":"<your-password>","cookie_secure":false,"storage":"kong"}' > portal_session_conf
+    ```
+3. Create secret:
+    ```
+    $ oc create secret generic kong-session-config -n kong --from-file=admin_gui_session_conf=<manager-config-filename> --from-file=portal_session_conf=<portal-config-filename>
+    ```
+{% endnavtab %}
+{% endnavtabs %}
+## Step 7. Prepare Kong's configuration file
+
+1. Create a `values.yaml` file based on the sample in the [Kong charts repository](https://github.com/Kong/charts/blob/master/charts/kong/values.yaml). This file sets all the necessary parameters for your Kong environment.
+
+2. Minimally, for setting up Kong Enterprise on Kubernetes, you will need to set the following parameters:
+
+    |Parameter      | Value                         |
+    |---------------|-------------------------------|
+    |`env.database` | `"postgres"` or `"cassandra"` |
+    |`env.password.valueFrom.secretKeyRef.name` | Name of secret that holds the super admin password. In the example above, this is set to `kong-enterprise-superuser-password`. |
+    |`env.password.valueFrom.secretKeyRef.key` | The type of secret key used for authentication. If you followed the default settings in the example above, this is `password`. |
+    |`image.repository` | The Docker repository. In this case, `kong-docker-kong-enterprise-edition-docker.bintray.io/ kong-enterprise-edition`. |
+    |`image.tag` | The Docker image tag you want to pull down, e.g. `"1.5.0.2-alpine"`. |
+    |`ingressController.enabled` | Set to `true` if you want to use the Kong Ingress Controller, or `false` if you don't want to install it. |
+
+3. (Optional) If not using the Kong Ingress Controller, remove the entire section of `Ingress Controller parameters`.
```

Comment:
Probably remove this step entirely: the toggle is covered already in (2) and does need to remain in place: removing content from values.yaml will use the default from the base values.yaml, which will enable the controller.

The rest of the section doesn't do anything if `ingressController.enabled: false`, so it's safe to leave in place--removing it is mostly cosmetic, and instructing users to remove it runs the risk that they'll remove the toggle setting also.